pkgs/applications/virtualization/nvidia-docker/default.nix

```diff
@@ -0,0 +1,80 @@
+{ stdenv, lib, fetchFromGitHub, callPackage, makeWrapper, buildGoPackage, runc
+, libelf, libcap, libseccomp, linuxPackages, glibc }:
+
+with lib; let
+
+  glibc-ldconf = (glibc.overrideAttrs (oldAttrs: {
+    # ldconfig needs help reading libraries that have been patchelf-ed, as the
+    # .dynstr section is no longer in the first LOAD segment. See also
+    # https://sourceware.org/bugzilla/show_bug.cgi?id=23964 and
+    # https://github.com/NixOS/patchelf/issues/44
+    patches = oldAttrs.patches ++ [ ../../../development/libraries/glibc/ldconfig-patchelf.patch ];
```

Comment:
Ah so the patch is from https://sourceware.org/bugzilla/attachment.cgi?id=11444&action=diff

You should then use `fetchpatch` instead of adding it as a file.

Reply:
This was the patch submitted because of this PR, but yes it's much better documented. Fixed.